modules/sdk/src/main/scala/org/tessellation/sdk/domain/seedlist/SeedlistEntry.scala

```diff
@@ -0,0 +1,55 @@
+package org.tessellation.sdk.domain.seedlist
+
+import cats.Order
+import cats.syntax.contravariant._
+import cats.syntax.option._
+
+import org.tessellation.schema.peer.PeerId
+import org.tessellation.schema.trust._
+
+import com.comcast.ip4s.IpAddress
+import fs2.data.csv._
+import fs2.data.csv.generic.semiauto.deriveRowDecoder
+
+case class SeedlistEntry(
+  peerId: PeerId,
+  ipAddress: Option[IpAddress],
+  alias: Option[String],
+  bias: Option[TrustValueRefined]
+)
+
+object SeedlistEntry {
+
+  type Alias = String
```

Comment:
You should probably use a `@newtype` there.

Type aliases differ from new types. Type alias as you did here is just a reference to `String`. Consider such example which leads to errors:

```scala
type Age = Long
type Speed = Long

val age: Age = 18
val speed: Speed = age // No compile error!
```

where with `@newtype`:

```scala
@newtype
case class Age(value: Long)
@newtype
case class Speed(value: Long)

val age: Age = Age(18)
val speed: Speed = age // Compile error because Age is not assiblable to Speed
```

Reply:
`Alias` has been changed to a case class.